packages/sdk-js/src/sticky-bucket-service.ts

```diff
@@ -170,7 +196,10 @@ export class ExpressCookieStickyBucketService extends StickyBucketService {
     }
     return doc;
   }
-  async saveAssignments(doc: StickyAssignmentsDocument) {
+  async getAssignments(attributeName: string, attributeValue: string) {
+    return await this.getAssignmentsSync(attributeName, attributeValue);
+  }
```

Comment:
These async wrappers around sync functions are going to be identical for every sync service.  Should we just add them to `StickyBucketServiceSync` class directly instead?